packages/server/src/api/index.ts

```diff
@@ -17,7 +17,10 @@ const userManagementUserUpdate = `${management}/user-update`;
 const userManagementMergeWithSocial = `${management}/merge-with-social`;
 const userManagementAddFacebook = `${management}/add-facebook`;
 const userManagementAddGoogle = `${management}/add-google`;
+const userManagementShelfInvites = `${management}/user-invites`;
 const shelfUserInvite = `${shelf}/user-invite`;
+const shelfPendingInvites = `${shelf}/pending-invites`;
+const shelfRevokeInvite = `${shelf}/revoke-invite`;
 const shelfUserInviteAccept = `${shelf}/user-invite-accept`;
```

Comment:
I called these `user-invite` and `user-invite-accept`, so would expect these to follow the same pattern like `user-invite-revoke`. If you don't like my naming you could change `user-invite-accept` instead, or at least put the new two after it, not between `user-invite` and `user-invite-accept`.